hummingbot/connector/exchange/hotbit/hotbit_rest_post_processors.py

```diff
@@ -0,0 +1,21 @@
+import abc
+import types
+from typing import Any
+
+from hummingbot.core.web_assistant.connections.data_types import RESTResponse
+
+
+class HotbitRESTPostProcessorBase(abc.ABC):
```

Comment:
are you sure that this class should be ABC? I think that should be a subclass of RESTPostProcessorBase

Reply:
I think you are right and fixed it on the latest patch.